spec/v2/index.bs

```diff
@@ -1699,6 +1626,160 @@ For each new ProductFootprint, the [=data owner=] MUST
 4. set <{ProductFootprint/precedingPfIds}> to the set of <{ProductFootprint/id}> of the 1 or more preceding ProductFootprints
 
 
+# Product Identifiers # {#product-identifiers}
+
+<div class=note>Non-normative</div>
+
+To exchange PCF data between organizations, it is necessary to identify the related product or material. Given [=data owners=] and [=data recipients=] do not always (or often) use the same identification schemes, commonly and uniquely identifying the same product is a challenge - especially at scale. Given this situation, organizations must perform laborious and manual “mapping” exercises to map their identifier(s) for a product to the identify their supplier can understand.
+
+This specification describes how the product ID URN (Uniform Resource Name) should be constructed in cases where no formal namespace for a given product identifier is defined. 
+
+This will not eliminate the need for a mapping process but will ease mapping identifiers with a common, easily understood structure. Further, this proposal ensures interoperability with industry-specific product identifiers.
+
+We recognize there are existing relevant namespaces and corresponding URN syntax specifications. These can either be IANA-registered namespaces (like `urn:ISBN`) or widely used standards like `urn:gtin`. When product identification based on one or more of these standards is applicable, the corresponding namespaces should be used.
+
+## Product Identifier URN’s ## {#product-identifier-urns}
+Each ProductID MUST be a URN as specified in [[RFC8141]]. Accordingly, every URN conforms to the following syntax:
+
+```
+urn:namespace:namespace-specific-string
+```
+
+In determining which URN namespace and corresponding syntax to use, the data owner SHOULD follow the reasoning below:
+
+1.	If an IANA-registered URN namespace exists and is applicable, this SHOULD be used. See IANA Registered Namespaces for existing specifications, like ISBN.
```

Comment:
We could link to this page as a reference (https://www.iana.org/assignments/urn-namespaces/urn-namespaces.xhtml)?